README.md

```diff
@@ -313,6 +315,14 @@ queryClient.setQueryData(
 
 ```
 
+### `createUseQueryOptions`
+
+A functional version of the options usually passed the underlying `useQuery` hook. This is useful when interacting with `useQueries` API or queryClient methods (like [ensureQueryData](https://tanstack.com/query/latest/docs/reference/QueryClient#queryclientensurequerydata), etc).
```

Comment:
The name stutters a bit, and I think it can be confusing that `createUseQueryOptions` is not relevant for `useQuery` from `@connectrpc/connect-query`. What do you think about naming the function `createQueryOptions()`? 

Can we explain that it returns the query options `queryKey`, `queryFn`, and `enabled` for the given RPC? I imagine this would give users a better understanding.

Reply:
Yeah, I think I can get behind that. Yeah it's not the most straight forward for sure.

Reply:
Thanks! Looks great to me.